### DISCUSSION TITLE
OpenAI、特斯拉功臣轉戰教育，設立 AI 原生學校「Eureka Labs」

### DISCUSSION THREAD

**Discussion by Tina-Peng-1566** [1101-GS4719-C/Python-Community-GS4719-C#2059] [General]
https://technews.tw/2024/07/18/andrej-karpathy-starting-an-ai-plus-education-company-called-eureka-labs/

文中，OpenAI 共同創辦人卡帕斯的話令我印象非常深刻，也很認同:
「我們怎樣才能獲得學習新事物的理想體驗？以學習物理為例，人們可以想像向物理學家費曼（Richard Feynman）學習高品質的課程教材，他一步步指導你。但充滿熱情、擅長教學、無限耐心且精通世界上所有語言的專家非常少，無法根據需求親自教導全球 80 億人。」
至此，卡帕斯決定成立「AI 原生」新型學校──Eureka Labs。

因為AI厲害到能寫論文、生成圖片，而遭到許多老師、教育單位的排斥，所以看這篇文章後，有種「終於有人敢光明正大的向全世界宣布生成式AI將與教育、學習結合」的感覺。
AI是能幫助學習、加速學習效率的好工具。最近在學程式語言Python的我更能深感認同:只要有疑問或不懂的程式碼就問ChatGPT，甚至可以請它幫你優化程式碼、請它用簡單一點的方式說明。
不論是學習什麼，在過程中能有一個能讓你不斷提問、幫你解惑的「人」，可以隨時隨地、勇敢提問，不怕問笨問題被取笑，是件超級幸運的事情，尤其是對初學者而言。

**Comment by kingsophy** [comment]
我十分認同文章的最後一句話。這陣子在學習python時ChatGPT真的幫了我很多的忙，有時不知道程式哪裡寫錯的時候就需要麻煩ChatGPT幫我進行解答，對我來說，ChatGPT更像是一位幾乎無所不知的老師，我也期待著未來ChatGPT的進步，讓我進而學習更多新知。

**Comment by ds0w0** [comment]
你提到的卡帕斯成立 Eureka Labs 的計畫確實令人期待。AI 在教育領域的應用，特別是像生成式 AI 這樣的技術，能夠為學習者提供更靈活、更高效的學習方式。

我完全同意你的觀點，AI 可以顯著降低學習門檻，提升學習效率。像 ChatGPT 這樣的工具，不僅可以即時解答問題，還能根據用戶的需求提供定制化的學習支援。對於像我們這樣的初學者來說，有一個隨時可問、耐心解答的「人」是非常珍貴的。

正如你在學習 Python 時所體驗到的，AI 不僅能幫助我們解決具體的技術問題，還能幫助我們更好地理解複雜的概念。這樣的技術進步無疑能讓學習變得更加輕鬆和高效。我也很期待未來 Eureka Labs 的發展，以及 AI 在教育領域的更多創新應用。

而我在回覆你的文章時，尤其是遇到辭窮的時候，GPT 能提供豐富的表達方式和精確的語言，使得我們能夠更清晰地傳達想法。好比現在AI 的這種輔助功能，不僅提升了溝通效果，也讓我能夠更加自信地進行書寫和表達我對你文章的看法。對於提高我們的表達能力和溝通技巧，無疑是非常有價值的。

**Comment by hunteryang0108** [comment]
AI在幫助基礎教育發展上，真的能提供非常多的幫助。普通的教師在面對學生提出的問題時，可能會因為了解的資料不足而無法回答。但是AI教師擁有即時且龐大的線上資源可以查閱，只要學生提出的問題有曾被記載在文獻中，他就能進行相對應的解說。這樣的科技對於那些喜歡求知的學生來說是莫大的助益，對教育產業更是突破性的發展。
